justfile

```diff
@@ -59,8 +59,8 @@ reseed:
 import-hn:
 	npx tsx import-hn.ts ./other/hn-data/*.json.gz
 
-import-society-library json userid:
-	npx tsx other/import-society-library-debatemap.ts {{json}} {{userid}}
+import-society-library json="other/society-library-openai-debatemap.json" username="developer":
```

Comment:
I would not hardcode the filename here, it publishes too much information.

Reply:
What if I just change it to "debatemap.json"?

My motive here is that I want to use the justfile to remember command lines for common tasks. If I have to pass a lot of arguments to the justfile target, then I will end up creating another file to hold the 'just' commandline.

Reply:
That sounds good to me